starter-dev-backend/routes/graphql/index.js

```diff
@@ -0,0 +1,19 @@
+import { graphqlHTTP } from 'express-graphql';
+import { buildSchema } from 'graphql';
+
+const schema = buildSchema(`
+  type Query {
+    hello(greeting: String!): String!
+  }
+`);
+
+const root = {
```

Comment:
Better name for this variable
```suggestion
const resolvers = {
```